### ISSUE TITLE
Supported monitor missing ("duplicate" serial numbers?)

### ISSUE THREAD

**Issue by coletdjnz** [davidhi7/ddcci-plasmoid#1]
This widget is working great, and is a nice improvement over the built-in control.

Issue: it appears to be detecting/showing controls for only two of my three monitors. The third monitor (Display 3 in below output) works with the built-in plasma display brightness control but is missing from this widget.

output of `ddcutil detect`:
```
Display 1
   I2C bus:  /dev/i2c-4
   DRM connector:           card0-DP-2
   EDID synopsis:
      Mfg id:               GSM - Goldstar Company Ltd (LG)
      Model:                E2442
      Product code:         XXXXX  (0xXXXX)
      Serial number:        XXXXXXXXXXXX
      Binary serial number: XXXXXX (0xXXXXXXXX)
      Manufacture year:     2013,  Week: 1
   VCP version:         2.1

Display 2
   I2C bus:  /dev/i2c-6
   DRM connector:           card0-HDMI-A-1
   EDID synopsis:
      Mfg id:               GSM - Goldstar Company Ltd (LG)
      Model:                E2342
      Product code:         XXXXX  (0xXXXX)
      Serial number:        
      Binary serial number: XXXXXXXX (0xXXXXXXXX)
      Manufacture year:     2011,  Week: 1
   VCP version:         2.1

Display 3
   I2C bus:  /dev/i2c-8
   DRM connector:           card0-DP-1
   EDID synopsis:
      Mfg id:               PHL - Philips Consumer Electronics Company
      Model:                PHL 243V7
      Product code:         XXXXX  (0xXXXX)
      Serial number:        
      Binary serial number: XXXXX (0xXXXXXXXX)
      Manufacture year:     2020,  Week: 30
   VCP version:         2.2
   ```

![image](https://user-images.githubusercontent.com/27879174/230853486-0118d158-a673-4ed3-92ec-94ed17ea4a2c.png)

Looking at this code snippet:
https://github.com/davidhi7/ddcci-plasmoid/blob/3c002d9822ce4534bff2ed2e14b4f8ed8da5eef8/backend/ddcci_plasmoid_backend/ddcci.py#L50-L56

I suspect this may have something to do with two of the monitors having no `Serial number`? (though they do both have a `Binary serial number`).

**Comment by davidhi7** [comment]
I just released version 0.1.3 which should fix this bug. To apply the fix, update the backend with the command `pip install -U ddcci_plasmoid_backend`. Thank you for your report.

**Comment by coletdjnz** [comment]
Can confirm it is fixed. Thanks!
